src/plugins/orangehrmRecruitmentPlugin/Api/VacancyAttachmentAPI.php

```diff
@@ -75,6 +75,30 @@ public function getValidationRuleForGetAll(): ParamRuleCollection
     }
 
     /**
+     * @OA\Post(
+     *     path="/api/v2/recruitment/vacancy/attachments",
+     *     tags={"Recruitment/Vacancy Attachments"},
+     *     @OA\RequestBody(
+     *         @OA\JsonContent(
+     *             type="object",
+     *             @OA\Property(property="vacancyId", type="integer"),
+     *             @OA\Property(property="comment", type="string"),
+     *             @OA\Property(property="attachmentType", type="integer"),
```

Comment:
can add `maxLength` and `maximum`
```suggestion
     *             @OA\Property(
     *                 property="comment",
     *                 type="string",
     *                 maxLength=OrangeHRM\Recruitment\Api\VacancyAttachmentAPI::PARAM_RULE_COMMENT_MAX_LENGTH
     *             ),
     *             @OA\Property(
     *                 property="attachmentType",
     *                 type="integer",
     *                 maximum=OrangeHRM\Recruitment\Api\VacancyAttachmentAPI::PARAM_RULE_ATTACHMENT_TYPE_MAX_LENGTH
     *             ),
```